appveyor.yml

```diff
@@ -13,6 +13,8 @@ environment:
 install:
   - ps: Install-Product node $env:nodejs_version
   - npm install
+  - npm install mocha-appveyor-reporter
```

Comment:
FYI, I had put `npm install mocha-appveyor-reporter` _before_ `npm install` on purpose. As it is right now, the reporter install happens after the `npm run build` task and get lost/clogged up with the rest. Nicer to keep the installs together I think (but really no big deal!).